packages/modules/cra-template-itwin-viewer/template.json

```diff
@@ -14,13 +14,13 @@
       "@bentley/imodeljs-markup": "^2.6.4",
       "@bentley/imodeljs-quantity": "^2.6.4",
       "@bentley/itwin-client": "^2.6.4",
-      "@bentley/itwin-viewer-react": "^4.0.0",
+      "@bentley/itwin-viewer-react": "^5.0.0",
```

Comment:
Does this not link with `rush.json` to the current version of the itwin-viewer-react?

Reply:
Well isnt this because we're using independent versioning in the repo?

Reply:
Ah yea, you're right.  I should have looked at it first :)

Reply:
yes. for the time being, I'd prefer to verify major upgrades....at least until we can be more confident about semver across the board for our dependencies